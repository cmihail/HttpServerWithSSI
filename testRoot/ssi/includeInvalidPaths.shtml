<html>
<head>
	<title>SSI Include</title>
</head>

<body>
<!--#include file="../testRoot/included.html" -->
<!--#include file="/home" -->
</body>
</html>
