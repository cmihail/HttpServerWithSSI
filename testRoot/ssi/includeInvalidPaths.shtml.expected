<html>
<head>
	<title>SSI Include</title>
</head>

<body>


</body>
</html>
